### DISCUSSION TITLE
channels_categories_color.css problem for discord

### DISCUSSION THREAD

**Discussion by bruh976** [community/community#135592] [Programming Help]
### Body

Hello, im not sure if its a right place to ask but i dont know where to seek help. 

As a tittle said i was looking for a way to make channel/categories colored on discord and i found some css here
https://github.com/LawOff/10CustomCSS/blob/main/scripts/channels_categories_color.css
but its doesnt work for me for some reason and it seems like a original author isnt acctive anymore so i cant ask him for help, im also not knowledgeable enough to repair it by myself so im looking for help.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by erencanbas** [comment]
Can u try this?

```
/* Channels & Categories Color: */
.channel-name {color: #f59abb !important;}
.icon, .container {color: #C62347 !important;}

/* Rainbow alternative */
.channel-name {
    animation: chromaYeet 5s linear infinite;
}
.icon {
    animation: chromaYeet 6s linear infinite;
}
.container {
    animation: chromaYeet 7s linear infinite;
}

@keyframes chromaYeet {
    from {
        color: #6666ff;
    }
    10% {
        color: #0099ff;
    }
    50% {
        color: #00ff00;
    }
    75% {
        color: #ff3399;
    }
    100% {
        color: #6666ff;
    }
}
```

**Reply by bruh976** [reply]
Sorry for late answer, unfortunattely it still doesn't work for me.

![obraz_2024-08-26_022727207](https://github.com/user-attachments/assets/a057d112-4a89-45b6-830e-f930e2f77f44)

**Reply by erencanbas** [reply]
BetterDiscord is a popular third-party tool that allows you to customize Discord’s interface with themes and plugins. You can use it to change the color of channel names by following these steps:

1. Install [BetterDiscord](https://betterdiscord.app/)
2. **Find or Create a Theme:** You can either download a pre-made theme that includes the color changes you want or create your own.
3. **Edit the CSS File:**

```
.containerDefault-7RImuF {
   color: #ff0000; 
}
```
